tests/tools/oecertdump/host/sgx_quote.cpp

```diff
@@ -71,25 +70,51 @@ void output_certificate(const uint8_t* data, size_t data_len)
             XN_FLAG_COMPAT,
             XN_FLAG_SEP_CPLUS_SPC | XN_FLAG_DUMP_UNKNOWN_FIELDS);
     BIO_free_all(input);
-    printf("\n");
 #endif
     OE_UNUSED(data);
     OE_UNUSED(data_len);
 }
 
-void output_certificate_chain(const uint8_t* data, size_t data_len)
+void decode_certificate_pem(FILE* file, const uint8_t* data, size_t data_len)
 {
 #if defined(__linux__)
+    X509* x509;
+    BIO* input = BIO_new_mem_buf(data, (int)data_len);
+    x509 = PEM_read_bio_X509(input, NULL, 0, NULL);
+    if (x509)
+        X509_print_ex_fp(
+            file,
+            x509,
+            XN_FLAG_COMPAT,
+            XN_FLAG_SEP_CPLUS_SPC | XN_FLAG_DUMP_UNKNOWN_FIELDS);
+    BIO_free_all(input);
+#endif
+    OE_UNUSED(data);
+    OE_UNUSED(data_len);
```

Comment:
```suggestion
#else
    printf("Certificate printing is not yet implemented.\n");
    OE_UNUSED(data);
    OE_UNUSED(data_len);
 #endif
```

Rationale: these OE_UNUSED lines were incorrect because for linux they *are* used (line 82 for example).
But for Windows they are correct that this function does nothing, not even print a message saying what it didn't do (which seems wrong to me).   If this PR is to "enable oecertdump to log endorsement buffer", either make sure it works on Windows, or update the PR title and make sure a separate issue is filed to track the Windows work and output a message about it like this.

This problem seems to stem from a cut and paste of lines 73 etc which was already problematic code, and I'd like to see the same fix there.

Reply:
Updated